.github/workflows/syntax.yaml

```
@@ -7,27 +8,30 @@ on:
     paths:
       - .github/workflows/syntax.yaml
       - '**.sublime-syntax'
-      - '**/syntax_test*'
+      - '**/syntax_test_*'
       - '**.tmPreferences'
   pull_request:
     paths:
       - .github/workflows/syntax.yaml
       - '**.sublime-syntax'
-      - '**/syntax_test*'
+      - '**/syntax_test_*'
       - '**.tmPreferences'
 
 jobs:
   syntax_tests:
     name: Syntax Tests (${{ matrix.build }})
     runs-on: ubuntu-latest
-    timeout-minutes: 15  # default is 6 hours!
+    timeout-minutes: 15
     strategy:
       matrix:
         include:
           - build: latest  # Latest dev build
+            packages: master
           - build: 4169  # Oldest stable version we intent to support
+            packages: v4186
```

Tests against stable builds should ideally use matching package revisions to ensure compatibility with unmodified releases.

```suggestion
            packages: binary
```

---

Under the circumstances, we'd need to move the supported version forward, then, and the bug would persist for the old versions anyway. I tried `binary`, but the name of the extended JSON context is not available in `4169`.

I didn't really want to mess with tag series, but I can if that's what we need for this.

---

I therefore got used to create dedicated branches for each supported version, such as st4169, st4180, ... . I sometimes even vendored required core syntaxes in branches for older ST releases to ship features.